_config.yml

```diff
@@ -36,6 +36,8 @@ global_nav:
     id: data
   - title: Updates
     id: updates
+  - title: BCDA Partial Adjudicated Claims
```

Comment:
Should this read as "Partially Adjudicated Claims"?
```suggestion
  - title: BCDA Partially Adjudicated Claims
```

Reply:
pls refer to the approved doc from CMS
https://confluence.cms.gov/x/2oJUMQ